{"uuid": "2855b5b8-8b7d-4d9f-9453-9fbdc65da6d9", "children": ["b526d471-b128-484e-93ef-bd8c87257a8b"], "befores": [{"name": "init_web_driver", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: unknown error: net::ERR_CONNECTION_REFUSED\n  (Session info: chrome=131.0.6778.140)\nStacktrace:\n\tGetHandleVerifier [0x010EEC13+23731]\n\t(No symbol) [0x0107C394]\n\t(No symbol) [0x00F5BE63]\n\t(No symbol) [0x00F59CD8]\n\t(No symbol) [0x00F4D938]\n\t(No symbol) [0x00F4F230]\n\t(No symbol) [0x00F4DB87]\n\t(No symbol) [0x00F4D527]\n\t(No symbol) [0x00F4D441]\n\t(No symbol) [0x00F4B8DB]\n\t(No symbol) [0x00F4BE2B]\n\t(No symbol) [0x00F5E85E]\n\t(No symbol) [0x00FDBE87]\n\t(No symbol) [0x00FC1E5C]\n\t(No symbol) [0x00FDB46E]\n\t(No symbol) [0x00FC1BF6]\n\t(No symbol) [0x00F93F35]\n\t(No symbol) [0x00F94EBD]\n\tGetHandleVerifier [0x013CF0D3+3039603]\n\tGetHandleVerifier [0x013E2DEA+3120778]\n\tGetHandleVerifier [0x013DB592+3089970]\n\tGetHandleVerifier [0x011843B0+635984]\n\t(No symbol) [0x01084DCD]\n\t(No symbol) [0x01082068]\n\t(No symbol) [0x01082205]\n\t(No symbol) [0x01074FD0]\n\tBaseThreadInitThunk [0x75A9FCC9+25]\n\tRtlGetAppContainerNamedObjectPath [0x775B809E+286]\n\tRtlGetAppContainerNamedObjectPath [0x775B806E+238]\n\n", "trace": "  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1139, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\_pytest\\fixtures.py\", line 890, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\test_cases\\conftest.py\", line 39, in init_web_driver\n    driver.get(get_data('url'))\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\event_firing_webdriver.py\", line 81, in get\n    self._dispatch(\"navigate_to\", (url, self._driver), \"get\", (url,))\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\event_firing_webdriver.py\", line 114, in _dispatch\n    result = getattr(self._driver, d_call)(*d_args)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 363, in get\n    self.execute(Command.GET, {\"url\": url})\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 354, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\moshiach\\Automation\\test_automation_final_project\\.venv\\Lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 229, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1735229250046, "stop": 1735229257993}], "start": 1735229250046, "stop": 1735229258237}